pkg/apis/broker/v1beta1/trigger_lifecycle_test.go

```diff
@@ -260,104 +260,120 @@ func TestTriggerConditionStatus(t *testing.T) {
 	tests := []struct {
 		name                     string
 		brokerStatus             *BrokerStatus
-		topicStatus              bool
-		subscriptionStatus       bool
+		topicStatus              string
+		subscriptionStatus       string
```

Comment:
It will be better if we can make the UT test code style consistent 
Instead of using `string` here, can we use `corev1.ConditionStatus` as we use for `subscriberResolvedStatus`

Reply:
Makes sense, done.